lua/vim_dadbod_completion/blink.lua

```diff
@@ -0,0 +1,67 @@
+---@type blink.cmp.Source
+local M = {}
+
+function M.new()
+  return setmetatable({}, { __index = M })
+end
+
+function M:get_trigger_characters()
+  return { '"', '`', '[', ']', '.' }
+end
+
+function M:enabled()
+  local filetypes = { "sql", "mysql", "plsql" }
+  return vim.tbl_contains(filetypes, vim.bo.filetype)
+end
+
+function M:get_completions(ctx, callback)
+  local cursor_col = ctx.cursor[2]
+  local line = ctx.line
+  local word_start = cursor_col
```

Comment:
Last char in the line is never checked due to while loop subtraction on the `word_start`. I suggest adding a `+1` to make sure it's checked
```suggestion
  local word_start = cursor_col + 1
```

For example, without this fix, if you do something like `select * from users.`, you will not get any users columns in autocompletion. Omni function needs to receive the input until the trigger character.